### DISCUSSION TITLE
Sprint planning - s2

### DISCUSSION THREAD

**Discussion by SanAlexis** [nyx-ei/animagi#9] [Announcements]
# Sprint planning of sprint 2
## Goals
The purposes of the sprint planning is to identify which stories could be achieve during the sprint. So we need to:
* size each story to provide its complexity and time slot;
* designate which of the stories we want to deliver.
## Modus operandi
* Identify each story.
* Suggest a complexity for the story using the [Fibonacci thumb rule](https://velocitycounts.com/2013/05/13/why-do-high-performing-scrum-teams-tend-to-use-story-point-estimation/#:~:text=1%20jumbo%20jet%3F-,Enter%20Fibonacci,-The%20fibonacci%20sequence)
* Suggest a duration (in days) for the story.
👍🏿 
## Stories concerned
* #1 
* #3 
* #4 
* #5

**Comment by arnoldKouatep** [comment]
hello

**Reply by SanAlexis** [reply]
Hello @arnoldKouatep
